+neurostim/+plugins/mcc.m

```diff
@@ -126,25 +127,42 @@ function afterExperiment(o)
         function digitalOut(o,channel,value,varargin)
             % digitalOut(o,channel,value [,duration])
             % Output the value to the digital channel
-            % o.digitalOut(0,unit8(2)) will write '2' to port A
-            % o.digitalOut(3,false) will set bit #3 to false.
+            % There are two modes here:
+            % 1) if VALUE is uint8, a full byte is written to port CHANNEL
+            % 2) if VALUE is logical, the single bit is set on line CHANNEL
+            % Currently Port0/A is hard-coded as input and Port1/B as
+            % output, so CHANNEL should be 1 (port B) or 9-16 (lines 1-8 in port B)

```

Comment:
That is no longer true with the DDir option?

Reply:
That's right. The lines 133-34 will be deleted.